static/app/bootstrap/processInitQueue.tsx

```diff
@@ -16,6 +16,30 @@ const COMPONENT_MAP = {
 };
 
 async function processItem(initConfig: OnSentryInitConfiguration) {
+  /**
+   * Allows auth pages to dymanically load and execute Google reCaptcha V3 JavaScript API
+   */
+  if (initConfig.name === 'recaptcha') {
+    const captchaElement = document.getElementById(
+      initConfig.element_id
+    ) as HTMLInputElement;
```

Comment:
You can avoid the cast and optional chaining in the rest of the method by returning/bailing if the element doesn't exist.

```typescript
if (!captchaElement) {
   return;
}
```

The check will help typescript narrow the type on `captchaElement` to `HTMLElement` instead of `HTMLElement | null`.